registers/dump/main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"flag"
+	"log"
+
+	"github.com/benlaurie/gds-registers/register"
```

Comment:
Is this a private repo? I get a 404 when I try to visit it.

Reply:
Fixed.